### ISSUE TITLE
Adding an option to not exit the watcher when there is an error

### ISSUE THREAD

**Issue by gavrochelegnou** [iamvdo/pleeease#51]
Hi, 

Every time my css is saved with an error inside, "pleeease watch" throws an error and exists.
The terminal window being hidden I often don't know this happened and don't understand why my css is not in sync anymore and I have to go back to the right terminal window and restart "pleeease watch"

Is there a way to show/log an error but not exit the watcher on parse error ?

Thanks a lot for the excellent tool !

**Comment by gavrochelegnou** [comment]
For the record I managed to find a workaround : 
in .pleeeaserc add safe:true to the sourcemaps :
Example :
```{
  "in": "src/*.css",
  "out": "all.min.css",
  "browsers": ["last 30 versions"],
  "sourcemaps": {
    "safe": true
  }
}```

As the sourcemaps is passed to postcss this arguments forces postcss to "clean" the CSS ... I'd rather have an error shown than a magically cleaned file but at least it's not stopping to watch anymore.

**Comment by iamvdo** [comment]
Fixed in pleeease-cli@3.2.7
Thanks for reporting.
